media/js/base/send-to-device.js

```diff
@@ -0,0 +1,239 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// create namespace
+if (typeof Mozilla === 'undefined') {
+    var Mozilla = {};
+}
+
+(function($) {
+    'use strict';
+
+    var SendToDevice = function() {
+
+        this.formLoaded = false;
+        this.formTimeout = null;
+        this.smsEnabled = false;
+
+        this.$widget = $('#send-to-device');
```

Comment:
Yup, for spying on method calls. It's a small price to pay for easier testing :+1: